docs/tutorials/tutorial-update-data.md

```diff
@@ -23,161 +23,236 @@ sidebar_label: Update existing data
   ~ under the License.
   -->
 
+:::info
 
-This tutorial shows you how to update data in a datasource by overwriting existing data and adding new data to the datasource.
+Apache Druid supports two query languages: [Druid SQL](../querying/sql.md) and [native queries](../querying/querying.md).
+This document describes the SQL language.
+
+:::
+
+Apache Druid stores data and indexes in [segment files](../design/segments.md) partitioned by time.
+Once segments are created, they cannot be modified.
+To update existing data, you must rebuild and republish the segments.
```

Comment:
This is not entirely true, since you can overshadow portions of a segment. (Overwrite a specific row)

Maybe in most cases, to updated data, you should overwrite the segment you want to change (it's only 1 step so rebuild & republish seems like too much detail for tutorial level). In some cases you can overshadow a portion of a segment using a smaller segment granularity than the original segment.

Reply:
Updated text.